selectors-4/deferred-for-level-5

```diff
@@ -127,3 +127,14 @@ The local link pseudo-class '':local-link''</h3>
 		What about if the document url has less than N segments,
 		and the link is same-page?
 		Should "null segments" count as matching, or not?
+
+<h3 id="custom-pseudo">Exposing custom state: the '':state()'' pseudo-class</h3>
```

Comment:
I'd suggest `custom-state` as the ID rather than `custom-pseudo`.

Reply:
I changed it to "custom-state"